com.endlessm.EknServices.json.in

```diff
@@ -11,6 +11,13 @@
         "--share=network",
         "--socket=session-bus"
     ],
+    "add-extensions": {
+        "com.endlessm.EknServices2.Extension": {
+            "directory": "build/eos-knowledge-services/1",
+            "bundle": true,
+            "autodelete": true
```

Comment:
We probably want no-autodownload seeing as this extension is not so handy for anyone else... :)